resteasy-core/src/main/java/org/jboss/resteasy/core/ResteasyDeploymentImpl.java

```diff
@@ -86,6 +86,7 @@
    protected String paramMapping;
    protected Map<String, Object> properties;
    protected boolean statisticsEnabled;
+   protected boolean contextProxyToImplementAllInterfaces;
```

Comment:
contextProxyToImplementAllInterfaces is never referenced.

Reply:
Good catch - thanks @ronsigal!